openid-4-verifiable-presentations-1_0.md

```diff
@@ -1205,6 +1209,10 @@ Mandatory user interaction before sending the request, like clicking a button, u
 
 Requests from the Wallet to the Verifier SHOULD be sent with the minimal amount of information possible, and in particular, without any HTTP headers identifying the software used for the request (e.g., HTTP libraries or their versions). The Wallet MUST NOT send PII or any other data that could be used for fingerprinting to the Request URI in order to prevent user tracking. 
 
+## Authorization Error Response with Wallet unavailable error code
+
+In the event that another component is invoked instead of the Wallet, the user MUST be informed and give consent before returning the `wallet_unavailable` Authorization Error Response to the Verifier.
```

Comment:
so we are relying on this `another component that is invoked` to return this error, right?

Reply:
Yes